### ISSUE TITLE
ideas for basic functionality

### ISSUE THREAD

**Issue by batik-303** [batik-303/germonnoma#1]

**Comment by batik-303** [comment]
@smukhitov @Ogy88 plese write your ideas in project kanban bord

**Comment by smukhitov** [comment]
What kind of website are you trying to build? Is this going to be an Apartment/House rental web-site? Can you please share the details of the project?

**Comment by batik-303** [comment]
> @batik-303 
> 
> What kind of website are you trying to build? Is this going to be an Apartment/House rental web-site like Airbnb? Can you please share the details of the project?

yes! i'm trying to build website like Airbnb. In Projects section of this Repo you can see more details
